docs/UserGuide.md

```diff
@@ -122,7 +123,7 @@ Examples:
 
 Finds persons whose names contain any of the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find KEYWORD [MORE_KEYWORDS]…​`
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
```

Comment:
On Line 130, it states only the name is searched - we changed this behavior to all details of a person (name, phone, email, roles), can we fix this?

Reply:
On Line 138, can we check that this image is consistent with our UI? If not can we add a new picture to replace?